apps/src/blockly/customBlocks/googleBlockly/mutators/procedureDefMutator.js

```diff
@@ -160,6 +161,7 @@ export const procedureDefMutator = {
     setBlockDescription(this, state['description']);
     this.doProcedureUpdate();
     this.setDeletable(state['initialDeleteConfig'] === false ? false : true);
```

Comment:
Does something like this make the intention of all the `state['initialEditConfig'] === false ? false : true` statements more explicit? 🤔  

```
function isExplicitlyFalse(value) {
    return value === false;
}
this.setDeletable(!isExplicitlyFalse(state['initialDeleteConfig']));
this.setEditable(!isExplicitlyFalse(state['initialEditConfig']));
this.setMovable(!isExplicitlyFalse(state['initialMoveConfig']));
this.setStatements_(!isExplicitlyFalse(state['hasStatements']));
```